### PR TITLE
Update install instructions

### PR DESCRIPTION
**Pull request by sandoche2k**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -5,19 +5,27 @@ This is the installation instructions for the Zonemaster CLI utility.
...
+> versions too. If you have any issue in installing the Zonemaster engine with
+> earlier versions, please send a mail with details to contact@zonemaster.net
+>
+
 
 ## Prerequisite
 
 Before you install the Zonemaster CLI utility, you need the
 Zonemaster Engine test framework installed. Please see the
 [Zonemaster Engine installation instructions](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md).
```
Comment (matsduf) [inline docs/installation.md]: cpan() or rather cpanm() is q prerequisite for these instructions.

File: docs/installation.md [hunk 2]
```diff
@@ -29,10 +37,10 @@ Zonemaster Engine test framework installed. Please see the
...
 3) Now you are ready to run the zonemaster-cli command:
 
-`zonemaster-cli example.com`
+`zonemaster-cli zonemaster.net`
 
 
-## <a name="FreeBSD"></a> Instructions for FreeBSD 10.1
+## <a name="FreeBSD"></a> Install dependencies for FreeBSD
 
 1) First install the Zonemaster Engine, following the instructions above.
```
Comment (matsduf) [inline docs/installation.md]: Remove this line. It is already covered by the prerequisite.

File: docs/installation.md [hunk 3]
```diff
@@ -49,7 +57,7 @@ Zonemaster Engine test framework installed. Please see the
     $ zonemaster-cli example.com
```
Comment (matsduf) [inline docs/installation.md]: On line 47 it says "still as root". Instead it should be "su" or "sudo" (after it has been decided which one to use in all installation documents for FreeBSD).

File: docs/installation.md [hunk 4]
```diff
@@ -58,6 +66,14 @@ First install the Zonemaster Engine, following the instructions above.
...
+## What to do next?
+
+In case if you want to use the engine from a web interface, you will have to install the
+*[Backend](https://github.com/dotse/zonemaster-backend/blob/master/docs/installation.md)*
+and the *[GUI](https://github.com/dotse/zonemaster-gui/blob/master/docs/installation.md)*.
+To use the engine from the *[API](https://github.com/dotse/zonemaster-backend/blob/master/docs/API.md)*,
+you will have to install the *Backend*.
+
 
 -------
```
Comment (matsduf) [inline docs/installation.md]: Please update the section below:

>Copyright (c) 2013, 2014, 2015, IIS (The Internet Infrastructure Foundation)  
>Copyright (c) 2013, 2014, 2015, AFNIC  

to

>Copyright (c) 2013-2016, IIS (The Internet Foundation in Sweden)  
>Copyright (c) 2013-2016, AFNIC

File: docs/installation.md (hunk 1 above)
Comment (sandoche2k) [inline docs/installation.md]: As mentioned earlier, cpan or cpanm is not a prerequisite, as it would have been already installed for the engine.

Comment (matsduf) [inline docs/installation.md]: I accept that. If we stick with cpan() I guess it is installed with Perl as in FreeBSD.

File: docs/installation.md (hunk 3 above)
Comment (matsduf) [inline docs/installation.md]: In FreeBSD, su is the standard way, i.e.

```
su root -c pkg install p5-MooseX-Getopt p5-Text-Reflow p5-Module-Install
```

**Pull request merged**